pkg/printers/humanreadable.go

```diff
@@ -95,26 +104,90 @@ func (h *HumanReadablePrinter) EnsurePrintHeaders() {
 // See validatePrintHandlerFunc for required method signature.
 func (h *HumanReadablePrinter) Handler(columns, columnsWithWide []string, printFunc interface{}) error {
```

Comment:
I'd prefer to keep the typed functions.  I don't think runtime.Object is better than interface{}.  And I want two different types - one that takes a writer, and one that generates a table.

Also, printers aren't necessarily required to be about runtime.Object.